script.js

```diff
@@ -36,11 +39,9 @@ function generateQuote() {
     });
   tweetBtn.addEventListener("click", shareTweet);
 
-  shareTweet();
-
   function shareTweet(event) {
     const twitterUrl =
-      "https://twitter.com/intent/tweet?text=${quoteText} -${quoteAuthor}";
+      `https://twitter.com/intent/tweet?text=${quoteText} -${quoteAuthor}`;
```

Comment:
Changed from double quotes (`"`) to backtick as that is required for using the `${var}` syntax in JS.

Reply:
Noted with many thanks. The below link is just a reference for myself.

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals](url)